src/gcloud_common_utils/blob_helper.py

```diff
@@ -54,3 +54,23 @@ def download_blob(bucket_name: str, source_blob_name: str, raw_file):
     blob.download_to_file(raw_file)
```

Comment:
```
blob.download_to_file()
```
also works on open filehandles, so currently we use memory buffers like this:
```
            with BytesIO() as raw_file:
                raw_file = download_blob(raw_file_bucket, raw_file_name, raw_file)
```
but this also works:
```
            with open('my_file.txt', 'r') as my_file:
                raw_file = download_blob(raw_file_bucket, raw_file_name, my_file)
```
The other function:
```
blob.download_to_filename()
```
works without opening the file yourself.